courses/machine_learning/deepdive2/structured/solutions/5_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>Did you write this, Ryan?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>This is applying the feature engineering that we did earlier in Gwendolyn's course. We do some slightly different feature engineering but want to reinforce again what we did with taxifare, but now with babyweight. This is that lab's FEATURE_CROSS signature description to see again in case students need a refresher (It would have been 2 courses and MANY labs ago).</p>

Reply:
Let's give her credit where it's due. Instead of copying her work, why not link to her labs?
Please do the same for any other narrative you've copied.

Reply:
<p>I fully intend to add a link to the taxifare feature engineering lab once the lab exists in the repo (so that it's not a dead link). There's just a placeholder text file in there now lol.</p><p> </p><p>Putting the function signature again in the notebook itself is good reinforcement without having to open another lab and search for the relevant cell.</p><p> </p><p>As for the note, it'll be something like "Remember, we used these feature engineering methods with BQML back in [x_feature_engineering_taxifare](../../feature_engineering/solutions/x_feature_engineering_taxifare.ipynb)."</p>

Reply:
I'd rather err on the side of not accidentally plagiarizing someone's work. Please write your own summary, and we'll add a todo in the tracker sheet to link to Gwendolyn's lab once they are live.

Reply:
<p>Done, thanks for the review!</p>

Reply:
I apologize, Ryan. I could have been more tactful in my review.
Still, I can't approve this material. The point of these labs are to be summaries, and they're not exactly summaries if the students have seen this material before. Thank you for adding the links. Please streamline the descriptions here and make sure the code can be ran through by a student within a couple days. One sentence per code block should suffice.